src/rest-server/docs/swagger.yaml

```diff
@@ -2090,14 +2090,19 @@ components:
           type: integer
         attemptCompletedTime:
           type: integer
+          nullable: true
         exitCode:
           type: integer
+          nullable: true
         exitPhrase:
           type: string
+          nullable: true
         exitType:
           type: string
+          nullable: true
         exitDiagnostics:
           type: object
+          nullable: true
           properties:
             diagnosticsSummary:
               type: string
```

Comment:
The `runtime` object in next line is also nullable.

Reply:
done